Custom Resnet9 || CIFAR10 || Apple

tiagoakle opened the pull request.

santiago-akle committed:
Add apple tsv and json

tiagoakle commented:
Hi @codyaustun anything I can do to help?

codyaustun commented on CIFAR10/train/apple_resnet9_8v100_pytorch.json:
```diff
@@ -0,0 +1,11 @@
+    "model": "Custom Resnet 9",
+    "hardware": "Tesla V100 * 8 GPU / 32 GB / 40 CPU",
+    "timestamp": "2019-12-11",
+    "misc": {} 
```
Can you add details about your training routine (e.g., learning rate schedule, batch size, optimizer, etc.) and versions for any other important libraries (e.g., Cuda)? For example, take a look at [David Page's original submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/master/CIFAR10/train/davidcpage_resnet9_1v100-ec2_pytorch.json).

codyaustun commented on CIFAR10/train/apple_resnet9_8v100_pytorch.json:
```diff
@@ -0,0 +1,11 @@
+    "author": "Santiago Akle Serrano, Hadi Pour Ansari, Vipul Gupta, Dennis DeCoste",
+    "authorEmail": "sakle@apple.com",
+    "framework": "Pytorch 1.1.0",
+    "codeURL": "https://github.com/apple/ml-cifar-10-faster",
```
Please specify the commit hash either in the `codeURL` or as a separate `commitHash` field

codyaustun commented:
> Hi @codyaustun anything I can do to help?

Sorry for the delay! Overall the submission looks good, but it would be nice to add more information to the JSON file.

santiago-akle committed:
Detail schedules in Json

Appended hyper-parameter schedules to json
Included Cuda, CuDNN and NCCL versions to json

tiagoakle commented:
I think this addresses the comments

codyaustun reviewed (approved):
Thanks! Looks good to me!
